index.css

```
@@ -39,20 +40,22 @@
     align-items: center;
     /* 세로에 대한 정렬 */
     cursor: pointer;
-    background-color: white ;
+    background-color: white;
     font-size: 20px;
     font-weight: bold;
 }
 
-.number{
+.number {
     background-color: seashell;
 }
 
-.zero {
-    width: 215%;
+.large {
+    display: grid;
+    grid-template-columns: 2fr 1fr 1fr;
 }
 
+
 /* hover는 마우스 올렸을 때 변화 효과 */
 .input_button:hover {
```

숫자에 마우스오버 했을 때 색깔이 안바뀌길래 이상하다 싶었는데,
여기를 아직 안고쳐서 그랬군요.

```suggestion
.input-button:hover {
```

그리고 이렇게 `.input-button:hover`같은 애는 `.input-button`에 대한 스타일 정의 바로 아래에다가 위치시켜주는 게 좋습니다. 그래야 이런 실수를 방지할 수 있어요.

그리고 vscode를 사용해서 코딩하시는 거라면 `change all occurrences vscode`라는 기능을 이용해서 한번에 여러개를 수정할 수 있습니다. 그걸 사용했다면 이런 실수 방지가 또 됩니다.(개발자로 일하려면 수작업을 최대한 없애야합니다. 한땀한땀 수정하다보면 분명히 실수가 나옵니다.)